.circleci/config.yml

```diff
@@ -572,24 +565,23 @@ jobs:
           command: |
             cat "./env-for-circle/${CIRCLE_BRANCH}.sh" >> $BASH_ENV
             source $BASH_ENV
-            echo "export REINDEX_FLAG=$(./scripts/dynamo/get-reindex-flag.sh $ENV)" >> $BASH_ENV
             echo "export MIGRATE_FLAG=$(./scripts/dynamo/get-migrate-flag.sh $ENV)" >> $BASH_ENV
```

Comment:
I think we would want this entire step to go away, the `reindex` step defined started on line 553.

Reply:
Hmm, I think _that_ `reindex` step needs to stay as that's the long-running wait for reindex to finish step that occurs after a migration. 

However, per your comment I did remove this step b75f006b55e15cb6ada8674e97e63c0d895f36b0.

I'm going to run this over on another environment to ensure everything is 🅰️ 👌

Reply:
I believe the long running step you mentioned is the `wait-for-reindex` which is the approval job defined in the workflows :) 

The `reindex` job will now run enable the `wait-for-reindex` cron and run the `reindex-dynamo-records` script as long as the `migrate` flag is false, I don't think we want that but I could be wrong!

Reply:
You are 100% correct, thank you!!! Doing some more test runs before requesting your re-review.